suite/tests/CMakeLists.txt

```diff
@@ -3136,6 +3136,7 @@ if (X64 AND NOT AARCH64) # FIXME i#1569: get working on AArch64
 endif (X64 AND NOT AARCH64)
 
 tobuild("security-common.TestAllocWE" security-common/TestAllocWE.c)
+tobuild(security-common.except-int2d security-common/except-int2d.c)
```

Comment:
Along the same lines, this should be Windows-only and x86-only.  That's most easily done by adding it inside an existing conditional: next to other security-win32 tests.